wcs-importer-exporter.php

```diff
@@ -61,10 +64,31 @@ public static function setup_importer() {
 	}
 
 	/**
+	 * This is to calculate last_payment when the last payment not set by import
+	 * this is done by taking next interval and removing twice.
+	 * Only change from 0 if it was an imported subscription
+	 *
+	 * @param $date
+	 * @param $subscription @class WC_Subscription
+	 * @param $timezone
+	 * @return $date or date of last payment calulation
+	 */
+	public static function last_payment_calculation($date, $subscription, $timezone ) {
+		if ($date == 0 && "importer" === $subscription->__get("created_via")) {
```

Comment:
`$date == 0` here should be changed to `0 == $date` so that it is a Yoda condition and conforms to the WP coding standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions